src/installation/guides/arm-devices/platforms.md

```diff
@@ -7,8 +7,8 @@ Raspberry Pi Foundation's kernel tree, which should enable all special
 functionality that isn't available with mainline kernels. The RPi kernel
 packages also have their own header packages, `rpi-kernel-headers`. These
 packages should be installed if you want to use any DKMS packages. Void ships
-`rpi-base` meta packages that depend on the relevant kernel and `rpi-firmware`
-packages, which together enable Wi-Fi and Bluetooth functionality.
+`rpi-base` meta-packages that depend on the relevant kernel, and `rpi-firmware`
```

Comment:
Is this comma correct? `rpi-base` depends on `rpi-kernel` _and_ `rpi-firmware`

Reply:
Ah! This is an example of me not being sure of the intended semantics. :-) Will change.

Reply:
Changed to:

> Void ships `rpi-base` meta-packages that install the relevant `rpi-kernel` and `rpi-firmware` packages.